### PR TITLE
ability to create a new post

### PR DESCRIPTION
**Pull request by thejmdw**
git fetch --all
git checkout `thejmdw-post-form`

click on `New Post` in the `NavBar`
fill out the form fields
see that you've created a new post

### CODE REVIEW

File: src/components/posts/PostForm.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+//   }
+
+  const handleAdd = (e) => {
+    setIsLoading(true)
+    // debugger
+    let newPost = {
+        user_id: userId,
+        category_id: 1,
+        title: post.title,
+        publication_date: 0,
```
Comment (gqgonzales) [inline src/components/posts/PostForm.js:70]: try:
`publication_date: new Date.now(),`

File: src/components/posts/PostForm.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+//     const newHouse = { ...house }
+//     newHouse.userTypeId = e.target.value
+//     setHouse(newHouse)
+//   }
+
+  const handleAdd = (e) => {
+    setIsLoading(true)
+    // debugger
+    let newPost = {
+        user_id: userId,
```
Comment (gqgonzales) [inline src/components/posts/PostForm.js:67]: try:
`user_id: parseInt(userId),`

**Review by gqgonzales — CHANGES_REQUESTED** [review]
Hello!

It's looking good, just need some minor tweaks on the keys of the newly generated post objects.

Check out the comments below for more specifics.

File: src/components/posts/PostForm.js (hunk 1 above)
Comment (thejmdw) [reply src/components/posts/PostForm.js]: resolved

File: src/components/posts/PostForm.js (hunk 2 above)
Comment (thejmdw) [reply src/components/posts/PostForm.js]: resolved

**Review by gqgonzales — APPROVED** [review]
Now it looks great!

**Comment by anmonzi** [comment]
Pulled this down and was able to create a post. Looks great!

**Pull request merged**